roundedimageview/src/main/java/com/makeramen/roundedimageview/Corners.java

```diff
@@ -0,0 +1,128 @@
+package com.makeramen.roundedimageview;
+
+import java.util.EnumSet;
+
+public enum Corners {
+
+  ALL(EnumSet.of(Corner.TOP_LEFT, Corner.TOP_RIGHT, Corner.BOTTOM_LEFT, Corner.BOTTOM_RIGHT)),
+  TOP_LEFT(EnumSet.of(Corner.TOP_LEFT)),
+  TOP_RIGHT(EnumSet.of(Corner.TOP_RIGHT)),
+  BOTTOM_LEFT(EnumSet.of(Corner.BOTTOM_LEFT)),
+  BOTTOM_RIGHT(EnumSet.of(Corner.BOTTOM_RIGHT)),
+  TOP(EnumSet.of(Corner.TOP_LEFT, Corner.TOP_RIGHT)),
+  BOTTOM(EnumSet.of(Corner.BOTTOM_LEFT, Corner.BOTTOM_RIGHT)),
+  LEFT(EnumSet.of(Corner.TOP_LEFT, Corner.BOTTOM_LEFT)),
+  RIGHT(EnumSet.of(Corner.TOP_RIGHT, Corner.BOTTOM_RIGHT)),
+  OTHER_TOP_LEFT(EnumSet.of(Corner.TOP_RIGHT, Corner.BOTTOM_LEFT, Corner.BOTTOM_RIGHT)),
+  OTHER_TOP_RIGHT(EnumSet.of(Corner.TOP_LEFT, Corner.BOTTOM_LEFT, Corner.BOTTOM_RIGHT)),
+  OTHER_BOTTOM_LEFT(EnumSet.of(Corner.TOP_LEFT, Corner.TOP_RIGHT, Corner.BOTTOM_RIGHT)),
+  OTHER_BOTTOM_RIGHT(EnumSet.of(Corner.TOP_LEFT, Corner.TOP_RIGHT, Corner.BOTTOM_LEFT)),
+  DIAGONAL_FROM_TOP_LEFT(EnumSet.of(Corner.TOP_LEFT, Corner.BOTTOM_RIGHT)),
+  DIAGONAL_FROM_TOP_RIGHT(EnumSet.of(Corner.TOP_RIGHT, Corner.BOTTOM_LEFT));
```

Comment:
i'm not sure why we need all these enum values. `Corners` should always have all corners with default radius of 0. Then we don't need the other `Corner` enum either.

Just need something like this right?
```
public class Corners() {
    float topLeft;
    float topRight;
    float bottomLeft;
    float bottomRight;
}
```

and maybe a helper `setAll(float radius)` function but that should be all